src/main/java/seedu/address/logic/commands/FeedbackCommand.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FEEDBACK;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Feedback;
+import seedu.address.model.person.Person;
+
+
+/**
+ * Edits the feedback details of an existing person in the address book.
+ */
+public class FeedbackCommand extends Command {
+
+    public static final String COMMAND_WORD = "fb";
```

Comment:
maybe command_word can be "feedback" and you can add command_alias to "fb" instead?

Reply:
Good catch, will edit accordingly